lib/utils.js

```diff
@@ -179,8 +179,8 @@ module.exports = {
         const invalidProperty = run.properties.find(prop => module.exports.getKeyName(prop) === 'invalid');
 
         return {
-          valid: validProperty && validProperty.value.type === 'ArrayExpression' ? validProperty.value.elements : null,
-          invalid: invalidProperty && invalidProperty.value.type === 'ArrayExpression' ? invalidProperty.value.elements : null,
+          valid: validProperty && validProperty.value.type === 'ArrayExpression' ? validProperty.value.elements.filter(Boolean) : null,
```

Comment:
I‘m considering changing the `null` to `[]` -- this can also avoid some null check.

Reply:
Sounds good to me.